pkg/generate/code/common.go

```diff
@@ -47,11 +49,14 @@ func FindIdentifiersInShape(
 	return identifiers
 }
 
-// FindIdentifiersInCRD returns the identifier field of a given CRD which
-// can be singular or plural. Errors iff multiple identifier fields detected
-// in the CRD.
+// FindIdentifiersInCRD returns the identifier fields of a given CRD which
+// can be singular or plural.
```

Comment:
I think it's worth noting in this docstring that this function returns the *original* member (field) name from the API model shape, not the renamed field name.